dialyzer.ignore-warnings

```diff
@@ -266,4 +266,16 @@ lib/websockex.ex:19: Function handle_cast/2 only terminates with explicit except
-The pattern 'error' can never match the type {#{'conn':='nil', 'log':='debug', 'path_params':=map(), 'pipe_through':=[any(),...], 'plug':='Elixir.OMG.WatcherRPC.Web.Controller.Account' | 'Elixir.OMG.WatcherRPC.Web.Controller.Challenge' | 'Elixir.OMG.WatcherRPC.Web.Controller.Fallback' | 'Elixir.OMG.WatcherRPC.Web.Controller.InFlightExit' | 'Elixir.OMG.WatcherRPC.Web.Controller.Status' | 'Elixir.OMG.WatcherRPC.Web.Controller.Transaction' | 'Elixir.OMG.WatcherRPC.Web.Controller.Utxo', 'plug_opts':=atom(), 'route':=<<_:48,_:_*8>>},fun((_,map()) -> any()),fun((_) -> map()),{'Elixir.OMG.WatcherRPC.Web.Controller.Account' | 'Elixir.OMG.WatcherRPC.Web.Controller.Challenge' | 'Elixir.OMG.WatcherRPC.Web.Controller.Fallback' | 'Elixir.OMG.WatcherRPC.Web.Controller.InFlightExit' | 'Elixir.OMG.WatcherRPC.Web.Controller.Status' | 'Elixir.OMG.WatcherRPC.Web.Controller.Transaction' | 'Elixir.OMG.WatcherRPC.Web.Controller.Utxo',atom()}}
\ No newline at end of file
+The pattern 'error' can never match the type {#{'conn':='nil', 'log':='debug', 'path_params':=map(), 'pipe_through':=[any(),...], 'plug':='Elixir.OMG.WatcherRPC.Web.Controller.Account' | 'Elixir.OMG.WatcherRPC.Web.Controller.Challenge' | 'Elixir.OMG.WatcherRPC.Web.Controller.Fallback' | 'Elixir.OMG.WatcherRPC.Web.Controller.InFlightExit' | 'Elixir.OMG.WatcherRPC.Web.Controller.Status' | 'Elixir.OMG.WatcherRPC.Web.Controller.Transaction' | 'Elixir.OMG.WatcherRPC.Web.Controller.Utxo', 'plug_opts':=atom(), 'route':=<<_:48,_:_*8>>},fun((_,map()) -> any()),fun((_) -> map()),{'Elixir.OMG.WatcherRPC.Web.Controller.Account' | 'Elixir.OMG.WatcherRPC.Web.Controller.Challenge' | 'Elixir.OMG.WatcherRPC.Web.Controller.Fallback' | 'Elixir.OMG.WatcherRPC.Web.Controller.InFlightExit' | 'Elixir.OMG.WatcherRPC.Web.Controller.Status' | 'Elixir.OMG.WatcherRPC.Web.Controller.Transaction' | 'Elixir.OMG.WatcherRPC.Web.Controller.Utxo',atom()}}
+apps/omg_child_chain/lib/child_chain.ex:35: Type specification 'Elixir.OMG.ChildChain':submit(transaction::binary()) -> {'ok',#{'txhash':='Elixir.OMG.State.Transaction':tx_hash(),'blknum':=pos_integer(),'txindex':=non_neg_integer()}} | {'error',submit_error()} is a subtype of the success typing: 'Elixir.OMG.ChildChain':submit(binary()) -> {'error',_} | {'ok',#{'blknum':=_, _=>_}}
+apps/omg_child_chain/lib/child_chain.ex:49: Type specification 'Elixir.OMG.ChildChain':get_block(hash::binary()) -> {'ok',#{'hash':=binary(),'transactions':=[any()],'blknum':=integer()}} | {'error','not_found' | 'internal_error'} is a subtype of the success typing: 'Elixir.OMG.ChildChain':get_block(binary()) -> {'error',_} | {'ok',#{'blknum':=_, _=>_}}
```

Comment:
Ignoring several "is a subtype of the success typing" as recommend by Ino. Let me know if that doesn't make sense here please.